### PR TITLE
feat: trivy template for slack

### PR DESCRIPTION
**Pull request by krol3**
A template for slack to support trivy output.

Here the result 

Screen Shot 2022-09-18 at 21 32 33

Thanks @AndreyLevchenko

### CODE REVIEW

File: rego-templates/vuls-trivy-slack.rego [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            "type": "mrkdwn",
+            "text": "*Found vulnerabilities*"
+        }
+    }
+	  ]
+
+    res := flat_array([
+        headers,
+        vln_list("CRITICAL"), 
+     #   vln_list("HIGH"),
```
Comment (krol3) [inline rego-templates/vuls-trivy-slack.rego:108]: @AndreyLevchenko I need to comment the vln_list, this is a [TODO]

**Comment by simar7** [comment]
Can you ask @AndreyLevchenko or his team to review this before we merge? Looks like some of these rules are duplicated and can be simplified.

**Comment by simar7** [comment]
Any update on this @krol3?

File: rego-templates/trivy-vulns-slack.rego [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+title = sprintf("Vulnerability scan report", []) # title is 
+
+aggregation_pkg := "postee.vuls.slack.trivy.aggregation"
+
+result = res {
+	severities := ["CRITICAL", "HIGH", "MEDIUM", "low", "negligible"]
+
+	headers := [
+		{"type": "section", "text": {"type": "mrkdwn", "text": sprintf("Image name: %s", [input.ArtifactName])}},
```
Comment (simar7) [inline rego-templates/trivy-vulns-slack.rego:128]:
```suggestion
		{"type": "section", "text": {"type": "mrkdwn", "text": sprintf("Artifact name: %s", [input.ArtifactName])}},
```

File: rego-templates/trivy-vulns-slack.rego [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+}
+
+###########################################################################################################
+postee := with_default(input, "postee", {})
+
+aqua_server := with_default(postee, "AquaServer", "")
+
+title = sprintf("Vulnerability scan report", []) # title is 
+
+aggregation_pkg := "postee.vuls.slack.trivy.aggregation"
```
Comment (simar7) [inline rego-templates/trivy-vulns-slack.rego:122]: is this used?

**Review by simar7 — CHANGES_REQUESTED** [review]
I tested it out and it looks great! just left a couple of nits and we can merge.

File: rego-templates/trivy-vulns-slack.rego (hunk 3 above)
Comment (krol3) [reply rego-templates/trivy-vulns-slack.rego:122]: yeah! It's by default this fields, it's used in the others templates, `title := input.image`

Comment (simar7) [reply rego-templates/trivy-vulns-slack.rego:122]: I understand, did you forget to check in the aggregate package then? Like this https://github.com/aquasecurity/postee/blob/f7fdfa7378778ace86a7a31f5f5e4b1ca4b8a404/rego-templates/vuls-slack-aggregation.rego#L1-L28

Comment (simar7) [reply rego-templates/trivy-vulns-slack.rego:122]: I don't see a `postee.vuls.slack.trivy.aggregation` rego package (file).

Comment (simar7) [reply rego-templates/trivy-vulns-slack.rego:122]: ping @krol3?

Comment (krol3) [reply rego-templates/trivy-vulns-slack.rego:122]: ok! make sense! I will added

**Review by simar7 — APPROVED** [review]
l(great)tm!

**Pull request merged**